km/km_filesys.c

```diff
@@ -1937,7 +1939,7 @@ static inline size_t fs_core_write_socket(char* buf, size_t length, km_file_t* f
    // Don't snapshot a socketpair with in flight data.
    if (file->how == KM_FILE_HOW_SOCKETPAIR0 || file->how == KM_FILE_HOW_SOCKETPAIR1) {
       if (ioctlfionread(fd) > 0) {
-         km_errx(1, "Couldn't perform snapshot, socketpair fd %d has queued data", fd);
+         km_warn("Couldn't perform snapshot, socketpair fd %d has queued data", fd);
```

Comment:
We probably don't want this change.
It is needed to get the node snapshot to work.